guides/operations/setup/installing_sqream_with_monit.rst

```diff
@@ -103,10 +105,559 @@ Note the following:
+         $ #SQREAM8-END
+         $ #       alert user@domain.com on {nonexist, timeout}
+         $ #                      with mail-format {
+         $ #                            from:     Monit@$HOST
+         $ #                            subject:  sqream2 $EVENT - $ACTION
+         $ #                            message:  This is an automate mail, sent from monit.
+         $ #                    }
+
+Starting Monit
+====================================  
+
+**To start Monit:**
+
+1. Stop all actively running SQream services:
+
+   .. code-block:: console
+     
+      $ sudo systemctl stop sqream[1-4]  #This command stops sqream1, sqream2,..,sqream4
+      $ sudo systemctl stop serverpicker
+      $ sudo systemctl stop metadataserver
+
+2. Start Monit as a super user:
+
+   .. code-block:: console
+     
+      $ sudo systemctl start monit
+   
+3. Verify that the following SQream processes are running and listening:
+  
+   .. code-block:: console
+     
+      $ sudo systemctl status metadataserver
+      $ sudo systemctl status serverpicker
+      $ sudo systemctl status sqream1  #... etc
+      $ sudo netstat -nltp   #to see that sqream is listening on ports  
+ 
+4. View Monit's service status:
+
+   .. code-block:: console
+     
+      $ sudo systemctl status monit
+
+5. If all good, enable the Monit service to start on boot: **Comment - What status result(s) are we looking for?**       
+       
+   .. code-block:: console
+     
+      $ sudo systemctl enable monit
+      
+Using Monit
+====================================
+
+Using Monit is simple and intuitive, and is used for starting processes that are not running and executing other actions in error situations. In addition, Monit has its own command syntax.
+
+For example, you can stop the **sqream3** service (being monitored by Monit) in one of the following ways:
+
+* Stopping Monit and SQream separately.
+* Stopping SQream using a Monit command.
+
+Both examples above are described in more detail below.
+
+Stopping Monit and SQream Separately
+-------------------------------------
+
+You can stop the Monit service and SQream separately as follows:
+
+.. code-block:: console
+     
+   $ sudo systemctl stop monit
+   $ sudo systemctl stop sqream3
+
+You can restart Monit as follows:
+
+.. code-block:: console
+
+   $ sudo systemctl start monit
+      
+Restarting Monit automatically restarts the SQream services.      
+
+Stopping SQream Using a Monit Command
+-------------------------------------
+
+You can stop SQream using a Monit command as follows:
+
+.. code-block:: console
+     
+   $ sudo monit stop sqream3
+
+This command spots SQream only (and not Monit).
+
+You can restart SQream as follows:
+
+.. code-block:: console
+     
+   $ sudo monit start sqream3
+       
+Monit Command Line Options
+-------------------------------------
+This section describes some of the most commonly used Monit command options:
```

Comment:
Best to show the command that shows the list

Reply:
What is the command?

Reply:
monit --help